lib/howl/ui/list_widget.moon

```diff
@@ -0,0 +1,287 @@
+-- Copyright 2012-2015 The Howl Developers
+-- License: MIT (see LICENSE.md at the top-level directory of the distribution)
+
+import Scintilla from howl
+import PropertyObject from howl.aux.moon
+import highlight, style, TextWidget, StyledTable from howl.ui
+import Matcher from howl.util
+
+append = table.insert
+
+style.define_default 'list_highlight', color: '#ffffff', underline: true
+
+highlight.define_default 'list_selection', {
+  style: highlight.ROUNDBOX,
+  color: '#888888'
+  alpha: 50
+  outline_alpha: 100
+}
+
+reversed = (list) -> [item for item in *list[#list, 1, -1]]
```

Comment:
In Python, `lst[::-1]` was often used to reverse in place, until they added a `reverse()` method.